home/migrations/0002_test_data.py

```diff
@@ -0,0 +1,42 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('home', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from django.contrib.auth.models import User
+        from home.models import Profile
+        user_test_data = [
+            ('Rebecca', 'RebeccaRebecca', 'Rebecca@gmail.com'),
+            ('Danit', 'DanitDanit', 'Danit@gmail.com'),
+            ('Aviv', 'AvivAviv', 'Aviv@gmail.com'),
+            ('Ido', 'IdoIdo', 'Ido@gmail.com'),
+        ]
+        profile_test_data = [
+            ('Rebecca', 'F'),
+            ('Danit', 'F'),
+            ('Aviv', 'M'),
+        ]
+
+        with transaction.atomic():
+            # Test: Create a User and Profile from a User via username.
+            for username, password, email in user_test_data:
+                User(username=username, password=password, email=email).save()
+            for username, gender in profile_test_data:
+                Profile(user=User.objects.get(username=username), gender=gender).save()
```

Comment:
While this will work, what I would suggest is to add the profile data into the user_test_data list, and process them together:
For example:
```
user_test_data = [
            ('Rebecca', 'RebeccaRebecca', 'Rebecca@gmail.com', 'F'),
            ('Danit', 'DanitDanit', 'Danit@gmail.com', 'F'),
            ('Aviv', 'AvivAviv', 'Aviv@gmail.com', 'M'),
            ('Ido', 'IdoIdo', 'Ido@gmail.com', None),
        ]
        with transaction.atomic():
            # Test: Create a User and Profile from a User via username.
            for username, password, email, gender in user_test_data:
                user = User(username=username, password=password, email=email).save()
                if gender:
                    Profile(user=user, gender=gender).save()

```

Reply:
Thank you for reviewing my PR. I'll change both files as you suggested.